optimum/exporters/openvino/model_patcher.py

```diff
@@ -2991,3 +2991,25 @@ def __init__(
     def __exit__(self, exc_type, exc_value, traceback):
         super().__exit__(exc_type, exc_value, traceback)
         self._model.forward = self._model.__orig_forward
+
+
+class InputEmbeddingPatcher(ModelPatcher):
+    def __init__(
+        self,
+        config: "OnnxConfig",
+        model: Union["PreTrainedModel", "TFPreTrainedModel"],
+        model_kwargs: Dict[str, Any],
+    ):
+        # making 16bit tracable overrides embeedings input signature these changes required to prevent this issue
```

Comment:
Could you please move the comment to where the patcher is called? Also, I don't quite understand what specifically in the original signature leads to problems, so rephrasing might help.

Reply:
do you mean on model configs level? in the moment patch applying, we can not identify which one of them used, only in config or in patcher class itself

Reply:
I mean here https://github.com/eaidova/optimum-intel/blob/ea/fix_16bit_embeddings/optimum/exporters/openvino/model_configs.py#L1271

Reply:
moved